### ISSUE TITLE
Issue - Retrobar crashes explorer constantly in windows 11 beta insider

### ISSUE THREAD

**Issue by ThaBluNate** [dremin/RetroBar#313]
I have been trying to deal with this problem for multiple hours now, after my entire os got bootlooped for this bug
Yes, that's right, this bug has bootlooped my pc entirely, and i have just now traced the bug to retrobar.
it was restarting explorer constantly, somehow corrupted some random shell file, and unlucky me had that shell file be essential to svchost.exe

WER logs:
Faulting application name: explorer.exe, version: 10.0.22581.200, time stamp: 0x44383e7a
Faulting module name: ucrtbase.dll, version: 10.0.22581.1, time stamp: 0xf5fc15a3
Exception code: 0xc0000409
Fault offset: 0x000000000007f61e
Faulting process id: 0x0x329C
Faulting application start time: 0x0x1D8485810E13427
Faulting application path: C:\WINDOWS\explorer.exe
Faulting module path: C:\WINDOWS\System32\ucrtbase.dll
Report Id: 298a51d8-075c-4c5b-af13-a0e7d4d0ff3b
Faulting package full name: 
Faulting package-relative application ID:

**Comment by dremin** [comment]
Is this still happening? I know some earlier Insider builds had explorer.exe loops that have since been fixed.

**Comment by ThaBluNate** [comment]
> Is this still happening? I know some earlier Insider builds had explorer.exe loops that have since been fixed.

I have had to stop using retrobar since then because it would bootloop me, so I don't know
Will reopen if it isn't fixed

**Comment by ThaBluNate** [comment]
Re-Opening - It's happened again on latest version,
Here's some eventvwr logs:

```
Faulting application name: explorer.exe, version: 10.0.22621.1, time stamp: 0xc721e923
Faulting module name: ucrtbase.dll, version: 10.0.22621.160, time stamp: 0xf5fc15a3
Exception code: 0xc0000409
Fault offset: 0x000000000007f61e
Faulting process id: 0x0x24F0
Faulting application start time: 0x0x1D8A61B824F7951
Faulting application path: C:\WINDOWS\explorer.exe
Faulting module path: C:\WINDOWS\System32\ucrtbase.dll
Report Id: 4c2bbe8b-5d7a-45f5-b72e-53e58ee042d0
Faulting package full name: 
Faulting package-relative application ID: 
```

I've looked it up and ucrtbase has something to do with .net (RetroBar uses .net so I quit it and stopped having explorer crash after about an  hour of running retrobar)

The only workaround has been signing out, which is annoying

**Comment by ThaBluNate** [comment]
Re-Opening - It's happened again on latest version,
Here's some eventvwr logs:

```
Faulting application name: explorer.exe, version: 10.0.22621.1, time stamp: 0xc721e923
Faulting module name: ucrtbase.dll, version: 10.0.22621.160, time stamp: 0xf5fc15a3
Exception code: 0xc0000409
Fault offset: 0x000000000007f61e
Faulting process id: 0x0x24F0
Faulting application start time: 0x0x1D8A61B824F7951
Faulting application path: C:\WINDOWS\explorer.exe
Faulting module path: C:\WINDOWS\System32\ucrtbase.dll
Report Id: 4c2bbe8b-5d7a-45f5-b72e-53e58ee042d0
Faulting package full name: 
Faulting package-relative application ID: 
```

I've looked it up and ucrtbase has something to do with .net (RetroBar uses .net so I quit it and stopped having explorer crash after about an  hour of running retrobar)

The only workaround has been signing out, which is annoying

**Comment by dremin** [comment]
@ThaBluNate I'm not seeing this on 22621.675, is this still happening for you?

**Comment by ThaBluNate** [comment]
Yea I had openshell while this was happening. I don't use either right now
so I don't have many more comments on this issue

**Comment by dremin** [comment]
Since you're not using it any more, I will close this issue, but I believe it will be fixed by the fix for #457.

**Comment by ThaBluNate** [comment]
> Since you're not using it any more, I will close this issue, but I believe it will be fixed by the fix for #457.

Re-opening this as I actually did start using it again, and I figured out how to trigger it since then.
Trigger by literally just restarting explorer while retrobar is active. That's it.
Basically it crashes explorer on startup, not with any other programs open, this is exclusively a retrobar issue.
Probably 'fixable' by just closing retrobar automatically if it's detected that explorer crashes, kind of like DWMBlurGlass does it.
